### DISCUSSION TITLE
RP2040 (Rev 2023a) I2C Eject Buttons

### DISCUSSION THREAD

**Discussion by JakeCarter** [ZuluSCSI/ZuluSCSI-firmware#366] [General]
Hey all. I have a question about the Rev 2023a of the RP2040. Does this board support the I2C eject buttons? According to https://github.com/ZuluSCSI/ZuluSCSI-firmware/discussions/353, the I2C has moved from J303 to J307. J307 is a 1x5 instead of a 2x5 so I'm trying to figure out which pins I need for each eject button (assuming 2 are still supported).

Thanks for any help here,

Jake

**Comment by aperezbios** [comment]
@JakeCarter yep, this is supported. Does your ZuluSCSI RP2040 have the QWIIC JST connector soldered down at J303? The initial production run of Rev 2023a RP2040 full sized boards did not. Either way, you can use the pin header at J307. The bottom of the board has the signals silk-screened on.

**Reply by JakeCarter** [reply]
I see on the back where `GND`, `SDA`, and `SCL` are marked. Thanks for pointing that out!

Mine doesn't have anything on J303. Just the solder pads. Out of curiosity, what can the QWIIC connector be used with for the ZuluSCSI?
